frontend/javascripts/oxalis/view/right-border-tabs/trees_tab/tree_hierarchy_view.tsx

```
@@ -1,6 +1,7 @@
 import { DownOutlined } from "@ant-design/icons";
 import { type Tree as AntdTree, type GetRef, type MenuProps, Modal, type TreeProps } from "antd";
 import { SimpleRow } from "dashboard/folders/metadata_table";
+import { useWkSelector } from "libs/react_hooks";
```

Biome (1.9.4)

[error] This import is unused.

Unused imports might be the result of an incomplete refactoring.
Safe fix: Remove the unused import.


(lint/correctness/noUnusedImports)